app/services/main_conversation/tools/websocket_utils/handle_websocket_connection.py

```
@@ -85,6 +85,14 @@ async def handle_websocket_connection(websocket: WebSocket):
             try:
                 raw_message: dict = await websocket.receive_json()
 
+                 # Handle heartbeat/ping messages
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4 (comment)

(E114)